ddtrace/contrib/dbapi/__init__.py

```diff
@@ -315,18 +339,3 @@ def _get_vendor(conn):
 
 def _get_module_name(conn):
     return conn.__class__.__module__.split(".")[0]
-
-
-def _propagate_dbm_context(dbspan, args):
-    # type: (...) -> Tuple[Tuple[Any, ...]]
-    dbm_comment = _database_monitoring._get_dbm_comment(dbspan)
-    if dbm_comment is None:
-        return args
-    # If dbm propagation supported and enabled the first arg must be a sql query or a procedure.
-    assert len(args) > 0 and isinstance(args[0], str)
```

Comment:
This assertion is being removed, is this correct?

Reply:
Yup with this change the assertion is not required. Previously the first argument had be a string now it can be overritten to be anything (ex: psycopg composable).